src/components/Documentation.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+    <section class="section" id="dochome">
+    <div class="columns is-mobile is-centered">
+      <div class="column is-three-quarters-tablet is-half-desktop has-text-centered">
+        <div class="block">
+          <h1 class="title is-1">Documentation</h1>
+          <p>Resources for using Pangeo Forge</p>
+        </div>
+      </div>
+    </div>
+    <div class="content" id="firststeps">
+        <h3 class="title is-3">First steps</h3>
+        <p>New to Pangeo Forge?  Start here!</p>
+        <ul>
+            <li><strong>What is Pangeo Forge?</strong> - Read more about what Pangeo Forge is and
+            how it works.</li>
+            <li><strong>Introduction Tutorial</strong> - Ready to code?  Walk through creating and
+            deploying your first data transformation pipeline.</li>
```

Comment:
```suggestion
            deploying your first data transformation recipe.</li>
```
My feeling is we should avoid "pipeline" in favor of "recipe", unless we are referring to the specific case of execution pipelines, e.g. in `base.py` [here](https://github.com/pangeo-forge/pangeo-forge-recipes/blob/c4867719c4866ec46436b5d53d95ceb470a6f72e/pangeo_forge_recipes/recipes/base.py#L79-L95). But most users of Pangeo Forge will not actually touch that part of the code.

Reply:
I have no attachment to "pipeline" so that part is fine.  I am hesitant to use the word "recipe" except when we really mean specifically a recipe.  My leaning is that for the intro tutorial we want to encapsulate more than just walking a user through recipe creation, we want to go all the way from idea to new transformed data.  @cisaacstern  What do you think of just removing the final word and writing _deploying your first data transformation_?

Reply:
I just dropped "pipeline" altogether, but we could of course change to "recipe" if that is more appropriate